Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:16
+
+LABEL maintainer="me@jqshuv.xyz"
+LABEL org.opencontainers.image.source https://github.com/ZynerOrg/xyter
+LABEL org.opencontainers.image.description "An multi-purpose discord.js bot."
+LABEL org.opencontainers.image.licenses=GPL-3.0-only
+
+
+WORKDIR /app
+
+LABEL maintainer="me@jqshuv.xyz"
+LABEL org.opencontainers.image.source https://github.com/ZynerOrg/xyter
```

Comment:
Why is this ``org.opencontainers.image`` here? Is this something standard used in Dockerfiles?

Reply:
It's if you're pushing an image to the ghcr.io container registry, but as I read you're pushing to docker hub so I could remove it.